xbmc.py

```diff
@@ -317,15 +315,24 @@ def executeJSONRPC(jsonrpccommand):  # NOSONAR
     json_method = json_data["method"]
     json_responses = os.environ.get("KODI_STUB_RPC_RESPONSES")
     if not json_responses:
-        raise ValueError("Could not find JSON Response folder. Use the environment variable KODI_STUB_RPC_RESPONSES to set one")
-
-    json_response = os.path.join(json_responses, "{}.json".format(json_method.lower()))
+        raise ValueError(
+            "Could not find JSON Response folder. Use the environment variable KODI_STUB_RPC_RESPONSES to set one")
+
+    path = "{}.json".format(os.path.join(os.path.join(json_responses, json_method.lower())))
+    if os.path.isfile(path):
+        with io.open(path, mode='r', encoding='utf-8') as fd:
+            stub_content = json.loads(fd.read())
+            if isinstance(stub_content, dict):
+                return json_data.dumps(stub_content)
```

Comment:
I actually found an issue. This should be `json.dumps()` will fix in the final version of 0.0.3

Reply:
Whoops, seems that I made this fuck up. Actually did not do a test run with the backwards compatible logic.